public/index.html

```diff
@@ -6,19 +6,36 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <!-- Change title -->
-    <title>Document</title>
+    <title>Landing Page</title>
+    <link rel="stylesheet" type="text/css" href="styles\styles.css" />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/js/bootstrap.bundle.min.js" integrity="sha384-JEW9xMcG8R+pH31jmWH6WWP0WintQrMb4s7ZOdauHnUtxwoG2vI5DkLtS3qm9Ekf" crossorigin="anonymous"></script>
     <!-- Add bootstrap CDN -->
-    <!-- Add Google font of your liking - Example uses Exo Google Font -->
-    <!-- Add stylesheet  -->
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Exo:wght@100&display=swap');
+    </style><!-- Add Google font of your liking - Example uses Exo Google Font -->
+
 </head>
 
 <body>
     <!-- Start of landing - Use bootstrap as needed to replicate landing page same as README -->
+    
+
         <div class="content">
-            <div class="box">
-                <!-- Add Text (h1, h4, p tags )-->
-                <!-- Add button with link to Resume - Upload to drive/dropbox -->
+            <div id="home"> 
+                <div class="box">
+                        <h1>Hey, I'm</h1>
+                        <h3>MARIO POIRE</h3>
+                        <br> <!-- Add Text (h1, h4, p tags )-->
+                        <form action="https://drive.google.com/file/d/1TZuzDv9CkTgCHUwfOh32rSfiqYZZL681/view?usp=sharing">
+                             <button class="button"><span>Resume </span></button>
+                        </form>
```

Comment:
no need to have a form for this. look into anchor tags `<a href="https://www.w3schools.com">`